mstm/__init__.py

```diff
@@ -51,99 +51,109 @@ def separate_exponent(num):
 
 def calc_scat_matrix(targ, inc, th_min, th_max, phi_min, phi_max, delete=True): # targ is target object, inc is incident object
```

Comment:
Add docstrings to every function, preferably before you write the function.  That way you know what the inputs and outputs need to be. We use the [numpy docstring format](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)(see [example](https://github.com/numpy/numpy/blob/master/doc/example.py)).